### DISCUSSION TITLE
Rpi pico question

### DISCUSSION THREAD

**Discussion by Mdawg09** [meshtastic/Discussions#75] [RP2040/RP2350]
Hello all, I have a question and it might be stupid and possibly already answered, but here it goes.

Does the rpi pico w work with Bluetooth or wifi yet or is it still only a USB serial connection that works?

Thanks, have a good day/night

**Comment by GUVWAF** [comment]
The Pico W works with Wi-Fi (as a station, so to connect to an access point), but not with Bluetooth (yet): https://meshtastic.org/docs/hardware/devices/raspberrypi/pico/

**Reply by rpsainio** [reply]
Another possibility with a Pico is to activate serial and proto-mode. Then it is possible connect directly with serial Adapter and give config-commands with meshtastic-tool

**Comment by antempus** [comment]
@GUVWAF 

I have mine setup as a Wi-Fi client and while I can connect with the mobile client I cannot connect to it with the Web Client like I do when it is Serial. 

Is there way to connect via a web ui client?

**Reply by GUVWAF** [reply]
No, currently not. The web UI uses HTTP instead of plain TCP like the Android client. The Pico firmware doesn't support HTTP yet.
You could try this desktop client, however: https://github.com/meshtastic/network-management-client
